.github/workflows/extra_tests.yml

```diff
@@ -0,0 +1,104 @@
+name: TIM-VX Backend
+
+on:
+  pull_request:
+    branches: [ 4.x ]
+    types: [ labeled, opened, synchronize, reopened ]
+
+
+jobs:
+  x86-simulator-build-test:
+    if: contains(github.event.pull_request.labels.*.name, 'category:dnn_timvx')
+    runs-on: ubuntu-20.04
+    container: yuentau/ocv_ubuntu:20.04
```

Comment:
Please add reference to Dockerfile as comment at least.

Reply:
Done.